plutus-core/testlib/PlutusCore/Generators/QuickCheck/Builtin.hs

```diff
@@ -122,6 +126,26 @@ instance ArbitraryBuiltin ByteString where
     arbitraryBuiltin = Text.encodeUtf8 <$> arbitraryBuiltin
     shrinkBuiltin = map Text.encodeUtf8 . shrinkBuiltin . Text.decodeUtf8
 
+instance ArbitraryBuiltin BLS12_381.G1.Element where
+    arbitraryBuiltin =
+      BLS12_381.G1.hashToGroup <$> arbitrary <*> pure Data.ByteString.empty >>= \case
+      -- We should only get a failure if the second argument is greater than 255 bytes, which it isn't.
+           Left err -> error $ show err
+           Right p  -> pure p
+    shrinkBuiltin _ = []
+
+instance ArbitraryBuiltin BLS12_381.G2.Element where
+    arbitraryBuiltin =
+      BLS12_381.G2.hashToGroup <$> arbitrary <*> pure Data.ByteString.empty >>= \case
+      -- We should only get a failure if the second argument is greater than 255 bytes, which it isn't.
+           Left err -> error $ show err
+           Right p  -> pure p
+    shrinkBuiltin _ = []
+
+instance ArbitraryBuiltin BLS12_381.Pairing.MlResult where
+    arbitraryBuiltin = BLS12_381.Pairing.millerLoop <$> arbitraryBuiltin <*> arbitraryBuiltin
+    shrinkBuiltin _ = []
+
```

Comment:
It looks like you didn't update the

```haskell
instance KnownKind k => Arbitrary (MaybeSomeTypeOf k) where
```

instance in this file in order to actually enable generation of the new built-in types.

We need to somehow throw a warning/error in this case, but not quite sure how, or make a test that fails, or something, it's way too easy to miss that.

Reply:
Oh, you're right.  Thanks for pointing that out.